stdlib/listLabels.mli

```diff
@@ -266,6 +266,13 @@ val filter : f:('a -> bool) -> 'a list -> 'a list
 val find_all : f:('a -> bool) -> 'a list -> 'a list
 (** [find_all] is another name for {!List.filter}. *)
 
+val filteri : (int -> 'a -> bool) -> 'a list -> 'a list
```

Comment:
Sorry I missed it in my first review, but here you need a label `~f:(int -> 'a -> bool)`.

Reply:
Fixed. :)